tests/library/browsercontext-fetch.spec.ts

```diff
@@ -797,11 +797,6 @@ it('should dispose when context closes', async function({ context, server }) {
   expect(error.message).toContain('Response has been disposed');
 });
 
-it('should throw on invalid first argument', async function({ context }) {
```

Comment:
Why was it removed?

Reply:
We don't generally check parameter types at runtime - we have 1000+ parameters and only check the type here.